source/TimePicker.js

```diff
@@ -419,9 +442,11 @@
 					fmtParams.locale = this.locale;
 				}
 				var merFormatter = new ilib.DateFmt(fmtParams);
-				var am = ilib.Date.newInstance({hour:10});
-				var pm = ilib.Date.newInstance({hour:14});
-				this.meridiems = [merFormatter.format(am), merFormatter.format(pm)];
+				if (merFormatter.meridiems == 'chinese') {
+					merFormatter.meridiems = 'default';
+				}
+
```

Comment:
Hi @graynorton, this part is locale-specific code which I referred in Jira comment.  
Chinese and Ethiopia are only two locale which has some meridiem states more than AM&PM.
However, in last year, we decided to not display whole meridiem unless there is strong request.

This time, Ethiopia asked to show every meridiem and Chinese did not.
So I need to use locale-specific code.